cmake/RepositoryDependenciesSetup.cmake

```diff
@@ -10,4 +10,77 @@ if (TPL_ENABLE_CUDA AND NOT Kokkos_ENABLE_Cuda_Relocatable_Device_Code)
   elseif (${PROJECT_NAME}_ENABLE_ShyLU_NodeTacho)
     message(FATAL_ERROR "ERROR: ${PROJECT_NAME}_ENABLE_ShyLU_NodeTacho=ON but TPL_ENABLE_CUDA='${TPL_ENABLE_CUDA}' AND Kokkos_ENABLE_Cuda_Relocatable_Device_Code='${Kokkos_ENABLE_Cuda_Relocatable_Device_Code}' which is not allowed!")
   endif()
-endif() 
\ No newline at end of file
+endif() 
+
+#########################################################################
+# STKBalance does not work with GO=INT or GO=UNSIGNED
+# Warn and disable it if it was set as a dependence of STK
+# Error out if it was explicitly requested by user
+
+IF ("${Tpetra_ENABLE_DEPRECATED_CODE}" STREQUAL "OFF")  # Remove this test when
+                                                        # Tpetra DEPRECATED
+                                                        # code is removed
+  SET(KDD_ENABLE_DEPRECATED OFF)  # Remove this line when DEPRECATED is removed
```

Comment:
what does "KDD" stand for?

Reply:
KDD == my initials.  Lower risk of conflict with other variables than "TMP_" or similar. :)